repository/git_test.go

```diff
@@ -65,3 +66,34 @@ func TestConfig(t *testing.T) {
 	err = repo.LocalConfig().RemoveAll("section.key")
 	assert.Error(t, err)
 }
+
+// checkStoreCommit creates a commit and checks if it has been signed.
+// See https://git-scm.com/docs/git-log#Documentation/git-log.txt-emGem
+// for possible signature status values.
+func (repo *GitRepo) checkStoreCommit(t *testing.T, expectedSignedStatus string) {
```

Comment:
This helper should not be part of the normal `GitRepo` as this is only for testing. What about `func checkStoreCommit(t *testing.T, repo *GitRepo, expectedSignedStatus string) {` ?